great_expectations/validator/validator.py

```diff
@@ -1422,7 +1422,7 @@ def batches(self) -> Dict[str, Batch]:
 
     @property
     def loaded_batch_ids(self) -> List[str]:
-        return self.execution_engine.loaded_batch_data_ids
+        return self._execution_engine.loaded_batch_data_ids
```

Comment:
Why the change from the public -> private version of this attribute?

Reply:
@Kilo59 This is important -- we strive to rely on private fields inside class as much as possible, thereby reducing the number of public methods only to the minimum that is necessary.  This cleanup step is toward this goal.  Thanks.

Reply:
I agree with using private methods/fields to limit the surface area of the public API.
But in this case, should we remove the public `execution_engine` property as well?
Maybe that's coming later.

Reply:
I'm on the same page with using private methods to limit the surface area of the public API.
Do we need the public version of this attribute at all?